Remotion/ObjectBinding/Web.Development.WebTesting/ScreenshotCreation/ScreenshotBocDateTimeValuePickerExtensions.cs

```diff
@@ -201,6 +201,7 @@ private static bool DatePickerContentLoaded (IWebDriver driver, string datePicke
       driver.SwitchTo().DefaultContent();
       var iframe = driver.FindElement (By.Id (datePickerID)).FindElement (By.TagName ("iframe"));
       driver.SwitchTo().Frame (iframe);
+

```

Comment:
That method's new, so the change should be flattened into the previous commit.

Reply:
This was a change request from @Stefan-Ilic , thats why it got its own commit. I did squash it now.